tests/tests_suite_test.go

```diff
@@ -357,6 +368,31 @@ func (s *existingSspStrategy) SkipIfUpgradeLane() {
 	skipIfUpgradeLane()
 }
 
+func (s *existingSspStrategy) waitForDataImportCrons() {
+	Eventually(func(g Gomega) {
+		for _, dataImportCronTemplate := range s.ssp.Spec.CommonTemplates.DataImportCronTemplates {
+			importCron := &cdiv1beta1.DataImportCron{}
+			importCronNamespacedName := types.NamespacedName{
+				Namespace: dataImportCronTemplate.Namespace,
+				Name:      dataImportCronTemplate.Name,
+			}
+			if importCronNamespacedName.Namespace == "" {
+				importCronNamespacedName.Namespace = internal.GoldenImagesNamespace
+			}
+
+			g.Expect(apiClient.Get(ctx, importCronNamespacedName, importCron)).To(Succeed(),
+				fmt.Sprintf("Could not get DataImportCron: %s", importCronNamespacedName.String()))
+
+			g.Expect(importCron.Status.Conditions).To(ContainElement(Satisfy(
+				func(condition cdiv1beta1.DataImportCronCondition) bool {
+					return condition.Type == cdiv1beta1.DataImportCronUpToDate &&
+						condition.Status == v1.ConditionTrue
+				},
+			)), fmt.Sprintf("DataImportCron %s is not up to date", importCronNamespacedName.String()))
```

Comment:
again not a big deal but gstruct can look nicer here:
https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-operator/resource/generate/rbac/controller_test.go#L44-L50

Reply:
Nice, I didn't know about `gstruct`, thanks.
I will update it in a future PR, because we want this fix merged soon.